protocols/bee.h

```diff
@@ -58,10 +58,11 @@ void bee_free( bee_t *b );
 /* TODO(wilmer): Kill at least the OPT_ flags that have an equivalent here. */
 typedef enum
 {
-	BEE_USER_ONLINE = 1,    /* Compatibility with old OPT_LOGGED_IN flag */
-	BEE_USER_AWAY = 4,      /* Compatibility with old OPT_AWAY flag */
-	BEE_USER_MOBILE = 8,    /* Compatibility with old OPT_MOBILE flag */
-	BEE_USER_LOCAL = 256,   /* Locally-added contacts (not in real contact list) */
+	BEE_USER_ONLINE = 1,       /* Compatibility with old OPT_LOGGED_IN flag */
+	BEE_USER_AWAY = 4,         /* Compatibility with old OPT_AWAY flag */
+	BEE_USER_MOBILE = 8,       /* Compatibility with old OPT_MOBILE flag */
+	BEE_USER_LOCAL = 256,      /* Locally-added contacts (not in real contact list) */
+	BEE_USER_SPECIAL = 262144, /* Compatibility with old OPT_SPECIAL flag */
```

Comment:
So I left some space there as padding, since the LOCAL flag is kind of a different piece of info than online/away/mobile. Otherwise, nothing special. I don't think any of these values is inherited from Gaim anymore, but I could be wrong.

The OPT_ ones, yeah, I sure hope (didn't grep for it) they're not needed anymore anyway, definitely don't define new ones.